### ISSUE TITLE
A good BASH prompt for Hyper Tab Icons?

### ISSUE THREAD

**Issue by ChristinWhite** [dfrankland/hyperterm-tab-icons#30]
I apologize if this is a bit off topic but I'm hoping it related enough to be useful. Has anyone created a good custom BASH prompt that will automatically set the title based on the task or process in a way that works well for displaying the appropriate Hyper Tab Icons?

**Comment by dfrankland** [comment]
Check out this Hyper issue: https://github.com/zeit/hyper/issues/1188#issuecomment-267301723

There's also many Stack Exchange threads:
*   https://unix.stackexchange.com/questions/104018/set-dynamic-window-title-based-on-command-input
*   https://superuser.com/questions/42362/gnome-terminal-process-name-in-tab-title
*   https://stackoverflow.com/questions/10546217/how-to-view-process-name-in-terminal-emulator-tab-or-title-bar

**Comment by ChristinWhite** [comment]
Thank you so much! I'd seen a couple of those but hadn't fully figured out what was going on the first time I saw them, the Hyper issue gave me the extra context understand what they were doing and how to adapt it with my configuration. 

I've been able to get around the command line okay for a while but getting this all setup is the first time I've really dug into shell scripts and BASH customization, it's been a bit of an information overload figuring things out, I really appreciate you pointing me in the right direction!
